web-examples/src/main/java/io/vertx/example/web/mongo/Server.java

```diff
@@ -0,0 +1,66 @@
+package io.vertx.example.web.mongo;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.example.util.Runner;
+import io.vertx.ext.mongo.MongoClient;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.handler.*;
+
+/**
+ * This is an example application to showcase the usage of MongDB and Vert.x Web.
+ *
+ * In this application you will see the usage of:
+ *
+ *  * JADE templates
+ *  * Mongo Client
+ *  * Vert.x Web
+ *
+ * The application allows to list, create and delete mongo documents using a simple web interface.
+ *
+ * @author <a href="mailto:pmlopes@gmail.com>Paulo Lopes</a>
+ */
+public class Server extends AbstractVerticle {
+
+  // Convenience method so you can run it in your IDE
+  public static void main(String[] args) {
+    Runner.runExample(Server.class);
+  }
+
+  @Override
+  public void start() throws Exception {
+
+    // Create a mongo client using all defaults (connect to localhost and default port) using the database name "demo".
+    final MongoClient mongo = MongoClient.createNonShared(vertx, config().put("db_name", "demo"));
```

Comment:
This should be shared - we don't want each verticle instance to have its own pool.